scripts/run_no_prep_tests.sh

```diff
@@ -5,4 +5,4 @@
   '-Dtest=test_locally.**.*Test' test \
   -DfailIfNoTests=false \
   --no-transfer-progress && \
-  if git status --porcelain | grep .; then git diff --no-pager; exit 1; fi
```

Comment:
I borrowed `git diff --no-pager` from a thread in stackoverflow.com but it doesn't work on my laptop, so I just corrected this time.